### PR TITLE
global: homepage removal and record detail addition

### LINKED ISSUE #20: templates: update record detail
@hachreak commented on [Wed Mar 09 2016](https://github.com/inveniosoftware/invenio-records-ui/pull/29)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

---

@hachreak commented on [Wed Mar 09 2016](https://github.com/inveniosoftware/invenio-records-ui/pull/29#issuecomment-194226746)

@tiborsimko 

---

@hachreak commented on [Wed Mar 09 2016](https://github.com/inveniosoftware/invenio-records-ui/pull/29#issuecomment-194278165)

ping @jirikuncar 

---

@jirikuncar commented on [Wed Mar 09 2016](https://github.com/inveniosoftware/invenio-records-ui/pull/29#issuecomment-194310751)

IMHO we should not update this template but instead override it in `invenio-marc21`.

---

@lnielsen commented on [Wed Mar 09 2016](https://github.com/inveniosoftware/invenio-records-ui/pull/29#issuecomment-194318452)

@jirikuncar Fully agree - didn't notice before that the PR was actually on records-ui

### PR DESCRIPTION
**Pull request by hachreak**
* Adds a template for marc21 record. (closes #20)

* Removes the homepage. (addresses inveniosoftware/invenio#3598)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
ping @jirikuncar

File: invenio_marc21/static/scss/invenio_marc21/marc21.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.marc21-record{
```
Comment (jirikuncar) [inline invenio_marc21/static/scss/invenio_marc21/marc21.scss]: missing license?

File: invenio_marc21/templates/invenio_marc21/detail.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
+{# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015, 2016 CERN.
```
Comment (jirikuncar) [inline invenio_marc21/templates/invenio_marc21/detail.html]: 2016 only

File: invenio_marc21/templates/invenio_marc21/detail.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+-#}
+{%- extends config.MARC21_BASE_TEMPLATE %}
+
+{%- block css %}
+  {{ super() }}
+  {% assets "invenio_marc21_marc21_css" %}
```
Comment (jirikuncar) [inline invenio_marc21/templates/invenio_marc21/detail.html]: `invenio_marc21_detail_css` ?

**Comment by jirikuncar** [comment]
It would be nice if `invenio_marc21/templates/invenio_marc21/detail.html` can be easily integrated with Invenio-Theme. Do we need the custom css block there?

File: invenio_marc21/static/scss/invenio_marc21/marc21.scss (hunk 1 above)
Comment (hachreak) [inline invenio_marc21/static/scss/invenio_marc21/marc21.scss]: done

**Comment by jirikuncar** [comment]
Why do we need custom CSS instead of showing integration with Theme in examples (or directly Invenio-Base)?

**Comment by hachreak** [comment]
ping @jirikuncar

**Pull request closed without merging**